src/handlers/events/me.ts

```diff
@@ -0,0 +1,299 @@
+
+    case 'worst':
+      if (worstExpls.length > 0) {
+        return [
+          [button(messages.me.reviewAbove(), 'review')],
+          [button(messages.me.goBack(), 'selfmade')],
+        ];
+      } else {
+        return [[button(messages.me.goBack(), 'selfmade')]];
+      }
+
+    case 'review':
+      const mappedExpls = worstExpls.map(({ key }) => [
+        button(key, ['review', key]),
+      ]);
+      return [...mappedExpls, [button(messages.me.goBack(), 'worst')]];
+
+    case 'likes':
+      const likedExpls = await db.getExlpsLikedByUser(user);
+
+      if (likedExpls.length <= EXPLS_IN_PAGE) {
+        return [[button(messages.me.goBack(), 'home')]];
+      }
+
+      const prevOffset = `${offset - EXPLS_IN_PAGE}`;
+      const nextOffset = `${offset + EXPLS_IN_PAGE}`;
+
+      return [
+        [
+          button(messages.me.prev(), ['likes', prevOffset]),
+          button(messages.me.next(), ['likes', nextOffset]),
+        ],
+        [button(messages.me.goBack(), 'home')],
+      ];
+
+    default:
+      return [
+        [
+          button(messages.me.madeByMe(), 'selfmade'),
+          button(messages.me.likedByMe(), 'likes'),
+        ],
+      ];
+  }
+}
+
+async function meText(user: number, page: string = 'home', offset?: number) {
+  switch (page) {
+    case 'selfmade':
+      return messages.me.whichBasis();
+
+    case 'likes':
+      const likedExpls = await db.getExlpsLikedByUser(
+        user,
+        EXPLS_IN_PAGE,
+        offset,
+      );
+
+      return explsList(likedExpls, page);
+
+    case 'review':
+      return messages.me.clickToRemove();
+
+    case 'best':
+    case 'worst':
+      const usersExpls = await db.getExlpsMadeByUser(user, page, EXPLS_IN_PAGE);
+
+      return explsList(usersExpls, page);
+
+    case 'home':
+    default:
+      return messages.me.exploreMore();
+  }
+}
+
+async function statsText(user: User) {
+  const amount = await db.getExlpCountByUser(+user.id);
+  const karma = await db.getUserKarma(+user.id);
+  const best = await db.getExlpsMadeByUser(+user.id, 'best', 1);
+
+  const helloText = _.isEmpty(user.username)
+    ? messages.me.helloStranger()
+    : messages.me.hello(user.username);
+
+  const statTexts = [[helloText, messages.me.stats(amount, karma)].join(' ')];
+
+  if (!_.isEmpty(best)) {
+    statTexts.push(messages.me.bestSoFar(best[0].key));
+  }
+
+  return statTexts.join('\n\n');
+}
+
+export async function meKeyboard(ctx: Context, page: string = 'home') {
+  const keyboard = await createKeyboard(
+    ctx.from!.id,
+    page,
+    ctx.session.meOffset,
+  );
+
+  return { inline_keyboard: keyboard };
+}
+
+export async function meStart(ctx: Context) {
+  const keyboard = await meKeyboard(ctx);
+  const texts = {
+    navigate: await meText(+ctx.from!.id, 'home'),
+    stats: await statsText(ctx.from!),
+  };
+
+  await ctx.telegram.sendMessage(+ctx.from!.id, texts.stats, {
+    parse_mode: 'HTML',
```

Comment:
I'm wondering should we use only one type of parse mode? [Elsewhere](https://github.com/majori/expl-bot/blob/74be828cb2d2bbe5b685b97393183fed17693de9/src/handlers/commands/resolve.ts#L28) we are using markdown